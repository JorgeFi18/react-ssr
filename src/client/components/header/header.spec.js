import React from 'react';
import renderer from 'react-test-renderer';
import Header from './header.component';

describe('Header component', () => {

    it('Logo test should match with snapshot', ()=> {
        const component = renderer.create(
            <Header />
        )

        const tree = component.toJSON();
        const [ figure ] = tree.children;

        expect(figure.children).toMatchSnapshot();
    });
});