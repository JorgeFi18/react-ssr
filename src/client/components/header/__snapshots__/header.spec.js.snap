// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component Logo test should match with snapshot 1`] = `
Array [
  "Marvel Api",
]
`;
